support/Sources/GraphQL.swift

```diff
@@ -192,6 +192,10 @@ public class GraphQL {
 			}
 			return key
 		}
+        
+        internal func childResponseObjectMap() -> [GraphQL.AbstractResponse]  {
```

Comment:
why are we introducing an abstract method that used to be defined at the protocol layer? why not leave as part of the protocol instead of having this one with the fatalError?

formatting nitpick: spaces instead of tabs here